src/Adapter/Form/ChoiceProvider/ResourcesChoiceProvider.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
```

Comment:
2018 or 2019, you need to choose 😛

Reply:
is there a script to update it?

Reply:
`bin/console prestashop:license:update`

Reply:
i think i cannot use it, it changes licenses for whole PrestaShop. :/ it should be done in another PR.